server/views/pages/offences.njk

```diff
@@ -0,0 +1,42 @@
+{% extends "./index.njk" %}
+{% from "govuk/components/warning-text/macro.njk" import govukWarningText %}
+{% block body %}
+  <div class="govuk-grid-row">
+    <div class="govuk-grid-column-three-quarters">
+      <h1>Offences</h1>
+      {% if courtCase.sentenceTerms.length == 0 %}
+        {{ govukWarningText({
+          text: "This person is currently being held on remand.",
+          iconFallbackText: "Warning"
+        }) }}
+      {% endif %}
+    </div>
+  </div>
+  <div class="govuk-grid-row">
+    <div class="govuk-grid-column-one-quarter">
+      {% if courtCase.sentenceTerms.length == 0 %}
+        <div class="hmpps-sidebar-single-link">
+          <h4 class="govuk-!-margin-top-0">Court cases and offences</h4>
```

Comment:
Shouldn't skip heading levels - i.e. going from `h1` to `h4` here. Should go in sequence through the page source order, h1, h2, h3, etc
Can use govuk heading styles to change the size if needed, e.g. `govuk-heading-m` or `govuk-heading-s`